### PR TITLE
Container Deployment Update

### PR DESCRIPTION
**Pull request by resmaa**
"how to" formatting of  Container deployment page

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1001#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | d84345a421a607d6ea60a312de08d3dec9866803 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1001/librarium_build/1597189543335825408) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 1]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
 
 <br />
 
-1. Log in to the Palette Dev Engine console
+# Prerequisite
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:22]: We also need an app profile so we should list it here and link to the how to create an app profile page.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 2]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:19]: We need to clean up the intro. We need to explain why someone would benefit from this. Take a look at how we did the other How-to's.

CleanShot 2022-11-28 at 07 18 35

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 3]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
...
-1. Log in to the Palette Dev Engine console
+# Prerequisite
 
+* [Spectro Cloud Palette account](https://www.spectrocloud.com/get-started/)
 
-2. Select the `App Profiles` from the left ribbon menu, click `+ New App Profile` and provide the following information to the app creation wizard.
+
+## How to Deploy Container to App Profile
+
+1. Login to the Palette Dev Engine console.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:29]:
```suggestion
1. Login to [Palette](/devx#quickstartwithpaletteappmode).
```
This changes the language to be the same as the other How-to's related to app profile.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 4]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
...
+
+
+2. Select the **App Profiles** from the left **Main Menu** and click on the **New App Profile** button at the top right handside of the main screen.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:32]:
```suggestion
2. Select **App Profiles** from the left **Main Menu** and click on the **New App Profile** button at the top right-hand side of the main screen. 
```

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 5]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
...
+
+## How to Deploy Container to App Profile
+
+1. Login to the Palette Dev Engine console.
+
+
+2. Select the **App Profiles** from the left **Main Menu** and click on the **New App Profile** button at the top right handside of the main screen. 
+
+
+3. Provide the wizard with the following information and click on **Next** button after you have filled out the information.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:35]:
```suggestion
3. Provide the wizard with the following information and click on the **Next** button after you have filled out the information.
```

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 6]
```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
...
 |         Parameter           | Description  |
 |-------------------------------|-----------------|
 |Application Profile Name | A custom name for the App Profile|
-|Description (optional)   | Description of the App Profile, if any |
+|Description (optional)   | Description of the App Profile, if any | 
 |Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|
 
 
-3. Select **Container Deployment** from the **Services** and provide the following information to the wizard:
+4. **Add Service**: From the available services select the **Container Deployment**, [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:46]:
```suggestion
4. **Add Service**: From the available services list, select **Container Deployment**. [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
```

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 7]
```diff
@@ -58,6 +71,20 @@ To add a container tier to Palette Dev Engine App Profile,
        argument: HOSTNAME
  
  
-4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:74]:
```suggestion
6. Click the **Review** button when you have filled out the information and are ready to conclude the wizard. You can add more services to the app profile if needed.
```

The last part of the sentence read very awkward.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 8]
```diff
@@ -58,6 +71,20 @@ To add a container tier to Palette Dev Engine App Profile,
        argument: HOSTNAME
  
  
-4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+
+
+## Validation
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:77]: Resma, the validation steps are not correct. The user should visit the Apps page from the Main Menu and check its status there.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 9]
```diff
@@ -58,6 +71,20 @@ To add a container tier to Palette Dev Engine App Profile,
        argument: HOSTNAME
  
  
-4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:75]: The next step should be to deploy the App Profile through the App page. In the App page, the user would deploy a new app by using the App Profile.

CleanShot 2022-11-28 at 07 34 53

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Several changes are requested as some steps are missing and others need to be modified.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md (hunk 2 above)
Comment (resmaa) [reply content/docs/04.5-devx/05-app-profile/02-container-deployment.md:19]: done!

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md [hunk 10]
```diff
@@ -15,31 +15,54 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Container Deployment
 
-To add a container tier to Palette Dev Engine App Profile,
+Palette App Mode supports the use of containers, a standard unit of software that packages code and all its dependencies to run applications quickly and reliably from one computing environment to another. They contain all the required executables, binary codes, libraries, and configuration files. As a result, containers are lightweight and portable with less overhead. To add a container tier to Palette Dev Engine App Profile follow the steps below.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:18]:
```suggestion
Palette App Mode supports the use of containers, a standard unit of software that packages code and all its dependencies to run applications quickly and reliably from one computing environment to another. Containers contain all the required executables, binary codes, libraries, and configuration files. As a result, containers are lightweight and portable with less overhead. To add a container tier to Palette Dev Engine App Profile, follow the steps below. 
```
Clarified who they is.

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏻

**Pull request merged**